### PR TITLE
Pazuzu-138 updated the registry connector according to new API

### PR DESCRIPTION
**Pull request by boure**
related to #137 and #138

### CODE REVIEW

**Comment by boure** [comment]
Mmmh I think I messed up my commits somehow.

Anyway only last commit should matter.

File: storageconnector/registryconnector.go [hunk 1]
```diff
@@ -76,22 +75,20 @@ func (store *registryStorage) GetFeature(name string) (shared.Feature, error) {
 
 // Use the given regex to return a list of FeatureMeta.
 // name		a regex used to filter out FeatureMeta
-// TODO issue #138 -> untested function !
 // TODO issue #138 -> update method with the new API
```
Comment (rafaelsbz) [inline storageconnector/registryconnector.go]: Isn't this TODO done? (going by the name of the PR)

Comment (boure) [inline storageconnector/registryconnector.go]: yes

**Comment by boure** [comment]
:+1:

**Pull request merged**